### DISCUSSION TITLE
Import from url

### DISCUSSION THREAD

**Discussion by mgscreativa** [adminerevo/adminerevo#74] [Q&A]
Hi! I need to run a MySQL query file with 10000+ updates. ¿Is it possible to use adminer through URL (because I need to run it with wget) to run that query file? And I mean, setting URL with, user, password, host and locally stored SQL file.

Thanks!

**Comment by nitodico** [comment]
Sure what have you tried and qhy wget and not curl?

**Comment by mgscreativa** [comment]
Hi! thanks a lot for the reply!

I have tried nothing, because as far as I can tell, the database credentials are sent through POST, maybe there's a way to set that up in a separate config file, like the one you create for plugins.

About CURL, the limitation is from my hosting provider, it allows CRON jobs to be run as wget requests only, no command line magic, just wget and the URL that must be run

![imagen](https://github.com/adminerevo/adminerevo/assets/1286632/bf4b6d53-3307-4b0f-87da-9aad5d803161)
